rust-lib/src/lib.rs

```diff
@@ -15,7 +14,7 @@ use serde::Deserialize;
 use rgb::lnpbp::bitcoin::OutPoint;
 
 use rgb::lnpbp::bp;
-use rgb::lnpbp::lnp::transport::zmq::{SocketLocator, UrlError};
+use rgb::lnpbp::lnp::transport::{Error as UrlError, ZmqSocketAddr};
```

Comment:
`transport::Error` is not a `UrlError`. Better to leave it as `transport::Error`